function-support.html

```diff
@@ -1195,12 +1211,12 @@ <h2 id="logic">Logic and Set Theory</h2>
 
 <h2 id="macros">Macros</h2>
 	<div class="indented">
-	<p>Macros can be defined either via  a KaTeX <a href="https://github.com/Khan/KaTeX#rendering-options">rendering options</a> or by <code>\gdef</code>. Available functions include:</p>
+	<p>Macros can be defined either via  a KaTeX <a href="https://github.com/Khan/KaTeX#rendering-options">rendering option</a> or by <code>\gdef</code>. Available functions include:</p>
 		<div class="indented">
 		<p><code>\mathchoice \TextOrMath \@ifstar \@ifnextchar \@firstoftwo \@secondoftwo \relax</code></p>
-		<p><code>@</code> is a valid character for commands, as if <code>\makeatletter</code> were in effect.</p>
 		</div>
 	</div>
+	<p><code>@</code> is a valid character for commands, as if <code>\makeatletter</code> were in effect.</p>

```

Comment:
I think you moved it too far out -- now it doesn't have enough indent. Should be in one level but not two.